code/modules/client/preferences_gear.dm

```diff
@@ -830,7 +845,7 @@ var/global/list/gear_datums = list()
 	path = /obj/item/clothing/suit/unathi/robe
 	cost = 1
 	slot = slot_wear_suit
-//	whitelisted = "Unathi" // You don't have a monopoly on a robe!
+	whitelisted = "Unathi"
```

Comment:
These are not wearability restrictions - these are loadout restrictions. It follows the pattern of all cultural clothing - tunnel cloaks, zhan-khazan furs, etc. Only individuals with a cultural excuse are permitted to bring them aboard, but otherwise anyone can physically wear then.